setup.sh

```diff
@@ -165,6 +166,27 @@ function cmd_output_manager()
   return "$?"
 }
 
+# KW used git to track saved configs from kernel-config-manager.
+# It changed so this function removes the unused .git folder from
+# the dot_configs_dir
+function remove_kernel_config_manager_legacy_git()
+{
+  local -r original_path="$PWD"
+
+  [[ ! -d "$dot_configs_dir"/.git ]] && return
+
+  if pushd "$dot_configs_dir"; then
```

Comment:
The problem with `pushd` is the fact that this command displays the folder path, which I think is unnecessary. Is it possible to make it quiet? If not, let's use the `cd` command.

Reply:
It's possible to redirect stdout to null. I changed the code to append `&>/dev/null` but if you think that using `cd` is more elegant i can change the code.